viewflow_extensions/management/commands/flow_graph.py

```diff
@@ -21,9 +18,6 @@ class Command(BaseCommand):
         positional arguments:
           flow_path             complete path to your flow, i.e. myapp.flows.Flow
 
-        optional arguments:
-          -s, --svg             create graph as svg file
```

Comment:
If you remove this argument, also remove it from the docstring above (I couldn't comment on that line as it is not changed, so I'm commenting here)